generators/chipyard/src/main/scala/config/ShuttleConfigs.scala

```diff
@@ -0,0 +1,32 @@
+package chipyard
+
+import org.chipsalliance.cde.config.{Config}
+
+//-----------------
+// Shuttle Configs
+//-----------------
+
+class ShuttleConfig extends Config(
+  new shuttle.common.WithNShuttleCores ++
+  new chipyard.config.AbstractConfig)
+
+class ShuttleCosimConfig extends Config(
+  new chipyard.harness.WithCospike ++                            // attach spike-cosim
+  new chipyard.config.WithTraceIO ++
```

Comment:
Nit: Comment missing here (or ignore these comments since they are put in other files)